### PR TITLE
feat(query-builder-container): add new component

### PR DESCRIPTION
**Pull request by kagnamazian**
reviewers : @usabilla/fe-chapter @Maruceau @galagmon 
Preview : https://static-staging.usabilla.com/stylabilla/query-builder-container/docs/section-components.html#kssref-components-querybuildercontainer

### CODE REVIEW

File: src/components/_query-builder-container.scss [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+//        fill="#9CACBA"
+//        fill-rule="nonzero"
+//      />
+//    </svg>
+//  </div>
+//</div>
+//
+//Styleguide Components.queryBuilderContainer.collapsed
+
+.#{$ns}query-builder__collapsed {
```
Comment (MLKiiwy) [inline src/components/_query-builder-container.scss]: uhm why not using a modifier format?

File: src/components/_query-builder-container.scss [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+//     <svg class="sb-icon sb-icon--s" version="1.1" viewBox="0 0 10 6">
+//       <path d="M5 2.414l3.293 3.293a1 1 0 0 0 1.414-1.414l-4-4a1 1 0 0 0-1.414 0l-4 4a1 1 0 0 0 1.414 1.414L5 2.414z"
+//         fill="#9CACBA" fill-rule="nonzero" />
+//     </svg>
+//   </div>
+// </div>
+//
+// Styleguide Components.queryBuilderContainer
+
+.kss-modifier__example {
```
Comment (MLKiiwy) [inline src/components/_query-builder-container.scss]: can you move styleguide specific, to here: https://github.com/usabilla/stylabilla/tree/master/docs-builder/kss-assets/components (like popover)

File: src/components/_query-builder-container.scss [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  .#{$ns}query-builder {
+    border: 1px solid;
+  }
+}
+
+.#{$ns}query-builder {
+  display: flex;
```
Comment (MLKiiwy) [inline src/components/_query-builder-container.scss]: I'm not sure if it's required on the upper container?

File: src/components/_query-builder-container.scss [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+}
+
+.#{$ns}query-builder__container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.#{$ns}query-builder__component {
+  width: 100px;
```
Comment (MLKiiwy) [inline src/components/_query-builder-container.scss]:
```suggestion
  width: sb-px2rems(100px);
```

File: src/components/_query-builder-container.scss [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+  border: 2px solid $sb-color-dark-3;
```
Comment (MLKiiwy) [inline src/components/_query-builder-container.scss]: it's temporary right?

**Comment by Maruceau** [comment]
Can the "Query builder container collapsed" have a 16-pixel line-height instead of 19 pixels?

File: src/components/_query-builder-container.scss (hunk 1 above)
Comment (kagnamazian) [reply src/components/_query-builder-container.scss]: yeah stupid. replacing it

File: src/components/_query-builder-container.scss (hunk 4 above)
Comment (kagnamazian) [reply src/components/_query-builder-container.scss]: yes I will replace it. But this component will be defined properly when we will build the query-builder component. We will not a pre-defined width like that.

File: src/components/_query-builder-container.scss (hunk 5 above)
Comment (kagnamazian) [reply src/components/_query-builder-container.scss]: yeah the entire component css is temporary. I will add it in the code

File: src/components/_query-builder-container.scss (hunk 3 above)
Comment (kagnamazian) [reply src/components/_query-builder-container.scss]: yes it is to encapsulate the query-builder--container and the icon.

**Pull request closed without merging**